src/main/java/am/ik/yavi/core/ConstraintContext.java

```diff
@@ -77,8 +61,8 @@ public String name() {
 	static ConstraintContext from(Function<String, ?> function) {
```

Comment:
Argument type can be changed to `Function<String, Attribute>`

Reply:
This factory method is for converting existing attributes classes like `HttpHeaders` to `ConstraintContext`.
So I don't think `Function<String, Attribute>` is helpful here.

Reply:
True, a little bit too fast with my remark.